config/webpack.config.prod.js

```diff
@@ -6,10 +6,11 @@ module.exports = {
   output: {
     filename: '[name].[hash].min.js'
   },
-  devtool: false,
+  devtool: '#source-map',
```

Comment:
Why the `#`? Are you sure it is necessary?

Reply:
That's actually a good question, I just copied it from the `dev` config. I'll experiment a bit more [with that](https://webpack.js.org/configuration/devtool/), looks like there are better options.

Reply:
You were correct, there's no need for the `#`. I also switched the dev sourcemaps to a faster version.